media/js/firefox/family/banner-init.es6.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import init from './banner.es6';
+
+init();
```

Comment:
**Question**: how come we split the logic for the banner and the initialization for the banner into two files? More curious than anything!

Reply:
This is unit testing prep, similar to https://github.com/mozilla/bedrock/blob/main/tests/unit/spec/pocket/mobile-nav.js#L33